# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264106489 18000
#      Thu Jan 21 15:41:29 2010 -0500
# Node ID bb97e11ce1bac41e4069bf01bad9ca5daacb8d23
# Parent  0522a65bcd56a3e64fd12b992e597f56f5e7fe8b
configure.ac: don't check for standard C headers

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,14 @@
+2010-01-21  John W. Eaton  <jwe@octave.org>
+
+	* configure.ac: Don't check for standard C headers (by now, I
+	think we can safely assume they exist).  Don't check for
+	varargs.h.
+
 2010-01-21  John W. Eaton  <jwe@octave.org>
 
 	* bootstrap.conf (gnulib_modules): Include sys_stat in the list.
 	* configure.ac: Don't check for sys/types.h or unistd.h.
 
 2010-01-21  John W. Eaton  <jwe@octave.org>
 
 	* bootstrap.conf (gnulib_modules): Include unistd in the list.
diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1451,22 +1451,22 @@ AC_CHECK_SIZEOF(long double)
 
 AC_HEADER_STDC
 AC_HEADER_DIRENT
 AC_HEADER_TIME
 AC_HEADER_SYS_WAIT
 
 ### C headers
 
-AC_CHECK_HEADERS(assert.h curses.h direct.h dlfcn.h fcntl.h float.h \
-  floatingpoint.h grp.h ieeefp.h inttypes.h limits.h locale.h memory.h nan.h \
-  ncurses.h poll.h pthread.h pwd.h stdint.h stdlib.h string.h sunmath.h sys/ioctl.h \
+AC_CHECK_HEADERS(curses.h direct.h dlfcn.h fcntl.h \
+  floatingpoint.h grp.h ieeefp.h inttypes.h locale.h memory.h nan.h \
+  ncurses.h poll.h pthread.h pwd.h stdint.h sunmath.h sys/ioctl.h \
   sys/param.h sys/poll.h sys/resource.h sys/select.h sys/stat.h \
   sys/time.h sys/times.h sys/utsname.h sys/utime.h termcap.h \
-  utime.h varargs.h)
+  utime.h)
 
 ### C++ headers
 
 AC_LANG_PUSH(C++)
 AC_CHECK_HEADERS(sstream)
 AC_LANG_POP(C++)
 
 have_termios_h=no
